CODEOWNERS

```diff
@@ -73,6 +73,9 @@ homeassistant/components/arest/* @fabaff
 homeassistant/components/asuswrt/device_tracker.py @kennedyshead
 homeassistant/components/automatic/device_tracker.py @armills
 homeassistant/components/axis/* @kane610
+homeassistant/components/*/arest.py @fabaff
```

Comment:
Why did you add this line? Merge issue?

Reply:
(Same as previous)
Yes indeed, I saw a couple of merge conflicts that I had to fix and I think I did something wrong there. Should be fixed in 708f9aa90836d53ae9f85f6ad17884a076dfcd37.